apps/protocols/tcp/mode/responsetime.pm

```diff
@@ -99,105 +93,98 @@ sub check_options {
     my ($self, %options) = @_;
     $self->SUPER::check_options(%options);
 
-    if (!defined($self->{option_results}->{hostname})) {
-        $self->{output}->add_option_msg(short_msg => 'Please set the hostname option');
-        $self->{output}->option_exit();
-    }
-    if (!defined($self->{option_results}->{port})) {
+    $self->{option_timeout} = 5;
+    $self->{option_packets} = 5;
+    if (defined($self->{option_results}->{port}) && $self->{option_results}->{port} =~ /(\d+)/) {
+        $self->{option_port} = $1;
+    } else {
         $self->{output}->add_option_msg(short_msg => 'Please set the port option');
         $self->{output}->option_exit();
     }
```

Comment:
You can simply check that an option is defined and not empty. You can of course check that's an integer but we don't got this far most of the time ...